### PR TITLE
Fixes buttons, removes Theme, moves from AVAssetsLibrary to PHPhotoLi…

### PR DESCRIPTION
**Pull request by antiquebeta**
# What does this PR do
This pull request fixes some issues I found with buttons maybe being too small, it removes the Theming files and favours iOS 13'/Apple' default light and dark mode (iOS 12 will be added back later), it also removes the ALAssetsLibrary deprecation by switching to PHPhotoLibrary, I've also added the prior fix to OpenVGDB which was using the wrong url for box art.

### Where should the reviewer start
I've never done this before so can't really help with where but most files in Provenance have been altered.

### How should this be manually tested
Running it in the simulator or device will show all changes.

### Any background context you want to provide
None.

### What are the relevant tickets
#1264 #1164 

### Screenshots (if appropriate)
[Game Library](https://twitter.com/Antique_Dev/status/1208745723006554114?s=20)
[Emulator (In-Game)](https://twitter.com/Antique_Dev/status/1208731759589773312?s=20)

### CODE REVIEW

File: Spotlight/IndexRequestHandler.swift [hunk 1]
```diff
@@ -100,15 +100,15 @@ public final class IndexRequestHandler: CSIndexExtensionRequestHandler {
...
+                    return CSSearchableItem(uniqueIdentifier: "com.antique.game.\(game.md5Hash)", domainIdentifier: "com.antique.game", attributeSet: game.spotlightContentSet)
                 } else {
                     return nil
                 }
             })
         #else
             let items: [CSSearchableItem] = results.flatMap({ (game) -> CSSearchableItem? in
                 if !game.md5Hash.isEmpty {
-                    return CSSearchableItem(uniqueIdentifier: "com.provenance-emu.game.\(game.md5Hash)", domainIdentifier: "com.provenance-emu.game", attributeSet: game.spotlightContentSet)
+                    return CSSearchableItem(uniqueIdentifier: "com.antique.game.\(game.md5Hash)", domainIdentifier: "com.antique.game", attributeSet: game.spotlightContentSet)
```
[the remark this answers is not in the document]
Comment (JoeMatt) [reply Spotlight/IndexRequestHandler.swift]: This should stay as 	com.provenance-emu.game

File: Spotlight/IndexRequestHandler.swift [hunk 2]
```diff
@@ -100,15 +100,15 @@ public final class IndexRequestHandler: CSIndexExtensionRequestHandler {
         #if swift(>=4.1)
             let items: [CSSearchableItem] = results.compactMap({ (game) -> CSSearchableItem? in
                 if !game.md5Hash.isEmpty {
-                    return CSSearchableItem(uniqueIdentifier: "com.provenance-emu.game.\(game.md5Hash)", domainIdentifier: "com.provenance-emu.game", attributeSet: game.spotlightContentSet)
+                    return CSSearchableItem(uniqueIdentifier: "com.antique.game.\(game.md5Hash)", domainIdentifier: "com.antique.game", attributeSet: game.spotlightContentSet)
```
[the remark this answers is not in the document]
Comment (JoeMatt) [reply Spotlight/IndexRequestHandler.swift]: com.provenance-emu.game

**Comment by JoeMatt** [comment]
This pull request is massive. Can you break it into a PR per feature please.

**Comment by antiquebeta** [comment]
Separating modifications to new branches as @JoeMatt suggested, thanks Tosh

**Comment by JoeMatt** [comment]
> Separating modifications to new branches as @JoeMatt suggested, thanks Tosh

Let me know if you need any help. We don't have a contributors guide yet, unfortunately (we've had so little outside help at this point it hasn't been much of an issue).

**Pull request closed without merging**